src/Controller/NphParticipantSummaryController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Controller;
+
+use App\Service\Nph\NphOrderService;
+use App\Service\Nph\NphProgramSummaryService;
+use App\Service\ParticipantSummaryService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/nph/participant")
+ */
+class NphParticipantSummaryController extends AbstractController
+{
+    /**
+     * @Route("/{participantid}", name="nph_participant_summary")
+     */
+    public function index(
+        $participantid,
```

Comment:
To be consistent could you use camel case for this? participantid => participantId